### PR TITLE
SP API Data Kiosk Scheduler Solution - Python implementation

### PR DESCRIPTION
**Pull request by chrisskhoury**
This pull request includes the following updates:

1. **Addition of Data Kiosk Well-Architected Scheduler Flow**
   - Implemented a scheduler flow adhering to the Well-Architected Framework for the Data Kiosk project.

2. **Upgrade of Data Kiosk Sample Solution to CDK**
   - Migrated the existing Data Kiosk sample solution to the AWS Cloud Development Kit (CDK) for improved infrastructure management and deployment.

3. **README Enhancements**
   - Fixed various issues in the README file.
   - Added better illustrations to improve clarity and usability.

### CODE REVIEW

File: use-cases/data-kiosk/code/java/pom.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.30</version>
+            <version>1.18.22</version>
```
Comment (mafge) [inline use-cases/data-kiosk/code/java/pom.xml:41]: What is the reason for this change?

File: use-cases/data-kiosk/app/sp-api-app-cdk/lib/sp-api-app-cdk-stack.ts [hunk 2]
```diff
@@ -0,0 +1,848 @@
...
+		const spapiCreateScheduleLambdaFunctionHandler = new cdk.CfnParameter(this, 'spapiCreateScheduleLambdaFunctionHandler', {
+		  type: 'String',
+          noEcho: true,
+		  description: "Handler of Creating the Scheduled Query Lambda Function"
+		});
+
+		const spapiDeleteScheduleLambdaFunctionHandler = new cdk.CfnParameter(this, 'spapiDeleteScheduleLambdaFunctionHandler', {
+		  type: 'String',
+          noEcho: true,
+		  description: "andler of Deleting the Scheduled Query Lambda Function"
```
Comment (leonardonakagawa) [inline use-cases/data-kiosk/app/sp-api-app-cdk/lib/sp-api-app-cdk-stack.ts:89]: Missing H in Handler
